server/spec/hypervisor_check_in_spec.rb

```diff
@@ -619,4 +619,122 @@ def create_virtwho_client(user)
     consumer_client = Candlepin.new(nil, nil, consumer['idCert']['cert'], consumer['idCert']['key'])
     return consumer_client
   end
+
+it 'should allow a single guest to be migrated and revoke host-limited ents' do
+    owner = create_owner random_string('test_owner1')
+    user = user_client(owner, random_string("user"))
+    virtwho = create_virtwho_client(user)
+    uuid1 = random_string('system.uuid')
+    guests = [{:guestId => uuid1}]
+
+    host_consumer = user.register(random_string('host1'), :hypervisor, nil,
+      {}, nil, owner['key'], [], [], nil, [], 'hypervisor_id_1')
+    new_host_consumer = user.register(random_string('host2'), :system, nil,
+      {}, nil, owner['key'], [], [], nil, [], 'hypervisor_id_2')
+
+    guest_consumer = user.register(random_string('guest'), :system, nil,
+      {'virt.uuid' => uuid1, 'virt.is_guest' => 'true'}, nil, owner['key'], [], [])
+    # Create a product/subscription
+    super_awesome = create_product(nil, random_string('super_awesome'),
+                            :attributes => { "virt_limit" => "10", "host_limited" => "true" }, :owner => owner['key'])
+    create_pool_and_subscription(owner['key'], super_awesome.id, 20)
+    consumer_client = Candlepin.new(nil, nil, host_consumer['idCert']['cert'], host_consumer['idCert']['key'])
+    new_consumer_client = Candlepin.new(nil, nil, new_host_consumer['idCert']['cert'], new_host_consumer['idCert']['key'])
+    guest_client = Candlepin.new(nil, nil, guest_consumer['idCert']['cert'], guest_consumer['idCert']['key'])
+
+    virtwho.hypervisor_check_in(owner['key'], get_host_guest_mapping('hypervisor_id_1', [uuid1]))
+
+    # consumer_client.update_guestids(guests)
+    pools = consumer_client.list_pools :consumer => host_consumer['uuid']
+    pools.length.should == 1
+    consumer_client.consume_pool(pools[0]['id'], {:quantity => 1})
+    new_consumer_client.consume_pool(pools[0]['id'], {:quantity => 1})
```

Comment:
Should new_consumer_client consume from the same pool that consumer_client does?